custom_components/econet300/sensor.py

```diff
@@ -41,7 +41,7 @@ class EconetSensorEntityDescription(SensorEntityDescription):
         icon="mdi:fan",
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
-        device_class=SensorDeviceClass.SPEED,
+        device_class=SensorDeviceClass.POWER_FACTOR,
```

Comment:
There was warning in logs Generic speed can be measure ft/s, in/d, in/h, km/h, kn, m/s, mph, mm/d but not in %